ui/app/templates/components/clients/dashboard.hbs

```diff
@@ -2,7 +2,7 @@
   <p class="has-bottom-margin-xl">
     This dashboard will surface OpenBao's client usage over time. Clients represent a user or service that has authenticated
     to OpenBao. Documentation is available
-    <DocLink @path="/vault/docs/concepts/client-count">here.</DocLink>
+    <DocLink @path="/docs/concepts/client-count">here.</DocLink>
```

Comment:
As a FYI @bozzo this page doesn't exist because we removed it in #194 -- this entire section on client counts can be removed from the UI. :-)

Reply:
done in #734